tests/foreman/ui/test_contenthost.py

```diff
@@ -139,6 +139,7 @@ def test_positive_end_to_end(session, default_location, module_repos_collection_
 
     :CaseImportance: Critical
     """
+    vm.run(r'subscription-manager repos --enable \*')
```

Comment:
Could we enable/override-to-enable the custom repos into the AK itself in setup_content, instead of doing it explicitly on host?

Reply:
hmm, yeah that would be better...lemme give it some attention today